.travis.yml

```diff
@@ -1,41 +1,44 @@
 language: python
 
 dist: xenial
-sudo: false
 
 matrix:
   include:
-    - python: 3.6
+    - python: 3.5
       env: TOX_ENV='flake8'
-    - python: 3.6
+    - python: 3.5
       env: TOX_ENV='isort'
     - python: 3.6
       env: TOX_ENV='docs'
-    - python: 3.6
+    - python: 3.5
       env: TOX_ENV='frontend'
-    # Django 1.11
-    - python: 3.4
-      env: DJANGO='dj111' SWAP='noswap'
+    # Django 2.2
     - python: 3.5
-      env: DJANGO='dj111' SWAP='noswap'
+      env: DJANGO='dj22' CMS='cms37' SWAP='swap'
```

Comment:
filer does not depend on django CMS, do we have to introduce it in the build matrix?